src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -100,33 +100,47 @@ namespace OpenRCT2::Platform
         }
     }
 
+    static std::string GetCurrentWorkingDirectory()
+    {
+        char cwdPath[PATH_MAX];
+        if (getcwd(cwdPath, sizeof(cwdPath)) != nullptr)
+        {
+            return cwdPath;
+        }
+        return std::string();
+    }
+
     std::string GetDocsPath()
     {
-        static const utf8* searchLocations[] = {
+        // clang-format off
+        const std::string prefixes[]{
+            Path::GetDirectory(Platform::GetCurrentExecutablePath()),
+            GetCurrentWorkingDirectory(),
```

Comment:
Windows version only really uses executable path:
https://github.com/OpenRCT2/OpenRCT2/blob/b8e163ffcfcc3bca2c2763ae97da429433675c46/src/openrct2/platform/Platform.Win32.cpp#L163-L166
I'm not sure if cwd should be included - especially if it's only included here and not in other places. I'd prefer fixing any packaging/install scripts that don't place docs in expected places rather than adapt the code here.